vnext/Microsoft.ReactNative/XamlUIService.cpp

```diff
@@ -50,4 +50,19 @@ void XamlUIService::DispatchEvent(
   return uiManagerProperty;
 }
 
+ReactPropertyId<xaml::XamlRoot> XamlRootProperty() noexcept {
+  static ReactPropertyId<xaml::XamlRoot> propId{L"ReactNative.UIManager", L"XamlRoot"};
```

Comment:
>propId [](start = 41, length = 6)

disclaimer: I don't know the convention yet, just observinig a disrepancy between the naming of the variable in the method above, following that patter would lead me to `rootProperty` instead of `propId`?

Reply:
but then `deviceinfomodule.cpp` just uses `prop`. I gues there is no convention  :)

---
In reply to: [434960388](https://github.com/microsoft/react-native-windows/pull/5115#discussion_r434960388) [](ancestors = 434960388)

Reply:
I originally had them named specifically, but since I was copy/pasting them around, I started naming them more generically.  Its a local var whose name I dont think really matters much.